src/NzbDrone.Core/HealthCheck/Checks/MountCheck.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System.Linq;
+using NzbDrone.Common.Disk;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Tv;
+
+namespace NzbDrone.Core.HealthCheck.Checks
+{
+    public class MountCheck : HealthCheckBase
+    {
+        private readonly IDiskProvider _diskProvider;
+        private readonly ISeriesService _seriesService;
+
+        public MountCheck(IDiskProvider diskProvider, ISeriesService seriesService)
+        {
+            _diskProvider = diskProvider;
+            _seriesService = seriesService;
+        }
+
+        public override HealthCheck Check()
+        {
+            var mounts = _seriesService.GetAllSeries()
+                                       .Select(series => _diskProvider.GetMount(series.Path))
+                                       .DistinctBy(m => m.RootDirectory)
```

Comment:
You'd need to select all root folders for the series first since every series path should be unique.

```
.Select(s => s.Path.GetParentFolder())
.Where(m => m != null)
.Distinct()
```

The `Where` being a safety check to ensure it's not null (in case someone added a series path at the top of a drive/root, _shudders_).